buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -247,7 +247,7 @@ public static int indexOf(ByteBuf needle, ByteBuf haystack) {
 
         // When the needle has only one byte that can be read,
         // the firstIndexOf method needs to be called
```

Comment:
This comment is not correct anymore... Should it be : `the firstIndexOf method can be used` ?

Reply:
Sure